test/multipliers/VotingStreakMultiplier.t.sol

```diff
@@ -0,0 +1,338 @@
+
+        castVote(testAccount);
+
+        // Verify multiplier was updated to multiplierIncrement
+        assertEq(multiplier.getMultiplyingFactor(testAccount), multiplier.multiplierIncrement());
+        assertEq(
+            multiplier.validUntil(testAccount),
+            yieldDistributor.lastClaimedBlockNumber() + 2 * yieldDistributor.cycleLength()
+        );
+
+        // Roll to the next cycle
+        cycleIterator++;
+        setUpForCycle(yieldDistributor, cycleIterator);
+
+        castVote(testAccount);
+
+        // Verify multiplier was updated to 2 * multiplierIncrement
+        assertEq(multiplier.getMultiplyingFactor(testAccount), 2 * multiplier.multiplierIncrement());
+
+        // Roll to the next cycle
+        cycleIterator++;
+        setUpForCycle(yieldDistributor, cycleIterator);
+
+        castVote(testAccount);
+
+        // Verify multiplier was updated to 3 * multiplierIncrement
+        assertEq(multiplier.getMultiplyingFactor(testAccount), 3 * multiplier.multiplierIncrement());
+    }
+
+    // when the account has voted in 4 subsequent cycles, the account's multiplier is equal to maxMultiplier
+    function test_multiplier_increment_after_4_subsequent_cycles() public {
+        VotingStreakMultiplier multiplier = setUpVotingStreakMultiplier();
+        address testAccount = setUpTestAccount();
+        uint256 cycleIterator = 0;
+        setUpForCycle(yieldDistributor, cycleIterator);
+
+        // Initial multiplier should be 0
+        assertEq(multiplier.getMultiplyingFactor(testAccount), 0);
+
+        castVote(testAccount);
+
+        // Verify multiplier was updated to multiplierIncrement
+        assertEq(multiplier.getMultiplyingFactor(testAccount), multiplier.multiplierIncrement());
+        assertEq(
+            multiplier.validUntil(testAccount),
+            yieldDistributor.lastClaimedBlockNumber() + 2 * yieldDistributor.cycleLength()
+        );
+
+        // Roll to the next cycle
+        cycleIterator++;
+        setUpForCycle(yieldDistributor, cycleIterator);
+
+        castVote(testAccount);
+
+        // Verify multiplier was updated to 2 * multiplierIncrement
+        assertEq(multiplier.getMultiplyingFactor(testAccount), 2 * multiplier.multiplierIncrement());
+
+        // Roll to the next cycle
+        cycleIterator++;
+        setUpForCycle(yieldDistributor, cycleIterator);
+
+        castVote(testAccount);
+
+        // Verify multiplier was updated to 3 * multiplierIncrement
+        assertEq(multiplier.getMultiplyingFactor(testAccount), 3 * multiplier.multiplierIncrement());
+
+        // Roll to the next cycle
+        cycleIterator++;
+        setUpForCycle(yieldDistributor, cycleIterator);
+
+        castVote(testAccount);
+
+        // Verify multiplier does not exceed maxMultiplier * multiplierIncrement
+        assertEq(multiplier.getMultiplyingFactor(testAccount), MAX_MULTIPLIER * MULTIPLIER_INCREMENT);
+    }
```

Comment:
These tests should be removed in favor of a fuzz test 

You can reference the two tests [here](https://github.com/BreadchainCoop/breadchain/blob/952757e85598922624ddf0d9f4bb363c04e72f25/test/YieldDistributor.t.sol#L321) for how you can turn a vanilla test into a fuzz test